roles/addons/nhc/readme.rst

```diff
@@ -0,0 +1,58 @@
+Node Health Checker
+-------------------
+
+Description
+^^^^^^^^^^^
+
+This role install and configure NHC.
+
+Instructions
+^^^^^^^^^^^^
+
+To set checks, simply create variable **nhc_checks** and provide all checks with
+check name as key and arguments of this check as value. For example:
+
+.. code-block:: yaml
+
+  nhc_checks:
+    check_hw_mem_free: 1mb
+    check_fs_mount_rw: -f /scratch
+    ...
+
+If you need to define multiple time the same check, but with different arguments,
+simply provide a list of arguments:
+
+.. code-block:: yaml
+
+  nhc_checks:
+    check_hw_mem_free: 1mb
+    check_fs_mount_rw:
+      - -f /scratch
+      - -f /home
+    ...
+
+You can force usage of a static file instead of a generated one by setting
+**nhc_use_template** to *false*. In this mode, the role will by default look for
+a file called *nhc.conf* in files folder of the role, and copy it to
+*/etc/nhc/nhc.conf* on target host.
```

Comment:
This is a good idea. However, this will force the user to create a file in the role directory. For people who manage their IaaC with a vcs, it means it won't be synchronized with their inventory (it's best practice to separate the inventory from the roles), so it will be hard to maintain. Can't you consider a variable `nhc_static_configuration` to define in the inventory?
If not defined, use the current template.
If defined, print the value of `nhc_static_configuration` to the file nhc.conf. It's similar to your proposal, with the advantage of keeping the content of the file in the inventory. (One can create a file nhc.yml with this parameter only)

Reply:
Got it, and agree. I will do that.